File: Python/1179.py
vetorPar = []
vetorImpar = []

for i in range(15):
  entrada = int(input())
  
  if abs(entrada) % 2 == 0:
    vetorPar.append(entrada)
  else: vetorImpar.append(entrada)


  if len(vetorPar) == 5:
    for b in range(5):
      print(f'par[{b}] = {vetorPar[b]}')
    vetorPar = []

  if len(vetorImpar) == 5:
    for b in range(5):
      print(f'impar[{b}] = {vetorImpar[b]}')
    vetorImpar = []

for i in range(len(vetorImpar)):
  print(f'impar[{i}] = {vetorImpar[i]}')

for i in range(len(vetorPar)):
  print(f'par[{i}] = {vetorPar[i]}')

File: C/1179.c
#include <stdio.h>

int main() {
    int vetorPar[5], vetorImpar[5];
    int parCount = 0, imparCount = 0;

    for (int i = 0; i < 15; i++) {
        int entrada;
        scanf("%d", &entrada);

        if (abs(entrada) % 2 == 0) {
            vetorPar[parCount] = entrada;
            parCount++;
            if (parCount == 5) {
                for (int j = 0; j < 5; j++) {
                    printf("par[%d] = %d\n", j, vetorPar[j]);
                }
                parCount = 0;
            }
        } else {
            vetorImpar[imparCount] = entrada;
            imparCount++;
            if (imparCount == 5) {
                for (int j = 0; j < 5; j++) {
                    printf("impar[%d] = %d\n", j, vetorImpar[j]);
                }
                imparCount = 0;
            }
        }
    }

    for (int i = 0; i < imparCount; i++) {
        printf("impar[%d] = %d\n", i, vetorImpar[i]);
    }

    for (int i = 0; i < parCount; i++) {
        printf("par[%d] = %d\n", i, vetorPar[i]);
    }

    return 0;
}
